nevergrad/functions/control/mujoco.py

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import gym
```

Comment:
Im not familiar with mujoco so tell me if I'm wrong: if I get it well, the mujoco code is directly included with gym right? So this would fail only if people do not have mujoco and try to instantiate say `Ant` or `Humanoid`? Do you know what would be the error message? If it's not explicit it may be worth catching and reraising stating that it needs mujoco

Reply:
It raises exception during ``gym.make`` when trying to load mujoco-related envs without mujoco library

> gym.error.DependencyNotInstalled: No module named 'mujoco_py'. (HINT: you need to install mujoco_py, and also perform the setup instructions here: https://github.com/openai/mujoco-py/.)

I think it is pretty clear error message :)

Reply:
definitely, thanks!